src/components/dialogs/input-hooks.js

```diff
@@ -273,6 +273,8 @@ export const useConnectivityValue = ({
     const [errorBusBarSection, setErrorBusBarSection] = useState();
     const intl = useIntl();
     const studyUuid = useSelector((state) => state.studyUuid);
+    const currentNode = useSelector((state) => state.currentTreeNode);
```

Comment:
I would be instinctively against using Redux Store inside our custom hooks. But maybe it's ok ?
We could keep passing Uuids as hooks props no ? Seems ok for me.